packages/slate/src/transforms/node.ts

```diff
@@ -729,7 +730,7 @@ export const NodeTransforms = {
 
         Transforms.liftNodes(editor, {
           at: range,
-          match: n => node.children.includes(n),
+          match: n => Element.isAncestor(node) && node.children.includes(n),
```

Comment:
does this fix something?

Reply:
Yes, `node` is not guaranteed to be an `Element` or `Editor`, and thus might not have a `children` property. Type-guarding it first as an Ancestor prevents attempts to to `node.children.includes` on a `Text` node